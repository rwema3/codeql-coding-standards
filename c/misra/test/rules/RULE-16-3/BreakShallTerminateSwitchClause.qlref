rules/RULE-16-3/BreakShallTerminateSwitchClause.ql