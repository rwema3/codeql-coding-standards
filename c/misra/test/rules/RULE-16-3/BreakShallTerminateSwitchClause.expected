| test.c:11:3:11:9 | case ...: | The switch $@ does not terminate with a break statement. | test.c:11:3:11:9 | case ...: | clause |
| test.c:14:3:14:9 | case ...: | The switch $@ does not terminate with a break statement. | test.c:14:3:14:9 | case ...: | clause |
| test.c:26:3:26:10 | default:  | The switch $@ does not terminate with a break statement. | test.c:26:3:26:10 | default:  | clause |
| test.c:45:3:45:10 | default:  | The switch $@ does not terminate with a break statement. | test.c:45:3:45:10 | default:  | clause |
